### PR TITLE
[4.0] Plugins using more namespaced classes Part 3 (Final)

### PR DESCRIPTION
**Pull request by anuragteapot**
### Summary of Changes
Switch to the namespaced classes

### Testing Instructions

Simple code review

### CODE REVIEW

**Comment by anuragteapot** [comment]
@C-Lodder please review changes.

**Comment by C-Lodder** [comment]
Everything seems to look ok from my end

File: plugins/user/profile/field/tos.php [hunk 1]
```diff
@@ -9,7 +9,14 @@
...
 
-JFormHelper::loadFieldClass('radio');
+use Joomla\CMS\Factory;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Form\FormHelper;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Associations;
+
+FormHelper::loadFieldClass('radio');
```
Comment (joomdonation) [inline plugins/user/profile/field/tos.php]: Please remove this line and replace code in the same file:

```php
class JFormFieldTos extends JFormFieldRadio
```

```php
class JFormFieldTos extends \Joomla\CMS\Form\Field\RadioField
```

File: plugins/user/profile/field/dob.php [hunk 2]
```diff
@@ -9,7 +9,12 @@
 
 defined('JPATH_PLATFORM') or die;
 
-JFormHelper::loadFieldClass('calendar');
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Form\FormHelper;
+use Joomla\CMS\Layout\FileLayout;
+
+FormHelper::loadFieldClass('calendar');
```
Comment (joomdonation) [inline plugins/user/profile/field/dob.php]: Please remove this line and replace the code

```php
class JFormFieldDob extends JFormFieldCalendar
```
with:
```php
class JFormFieldDob extends \Joomla\CMS\Form\Field\CalendarField
```

**Comment by joomdonation** [comment]
Almost perfect. Just small things left as I commented. Also, look like you forgot to work on PlgInstallerFolderInstaller (plugins/installer/folderinstaller/folderinstaller.php), please work on that only remaining plugin as well

**Comment by anuragteapot** [comment]
@joomdonation done! 
I have already worked on  <code>plugins/installer/folderinstaller/folderinstaller.php</code>  but file is included in part2.
Next time I try to optimize these things.
Thanks :)

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 0f3c6a106695baa71f1e897e5dacd04f64ab6a1d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19324">issues.joomla.org/tracker/joomla-cms/19324</a>.</sub>

**Comment by mukul21** [comment]
here is a simple example to use namespace[ gostream](https://www.gomovies.lol "")
the following syntax may help you

<?php

namespace app\a{
    class one{
       public static function _1(){
        echo 'a one _1<br>';
       }
    }
}

namespace app\b{
    class one{
        public static function _2(){
            echo 'b one _2<br>';
        }
    }
}

namespace app{

    echo a\one::_1();
    echo b\one::_2();
    echo a\two::_1();
}

namespace app\a{
    class two{
       public static function _1()){
        echo 'a two _1<br>';
       }
    }
}

prints 
a one _1
b one _2
a two _1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19324">issues.joomla.org/tracker/joomla-cms/19324</a>.</sub>

**Comment by Anu1601CS** [comment]
@mukul21 what you want to say?

**Comment by anuragteapot** [comment]
@mbabker sorry, if I comment something wrong. I don't know what he was trying to say.

**Comment by mbabker** [comment]
You didn't, it was spam.

**Comment by anuragteapot** [comment]
@mbabker Ok, Thanks
By the way, can you check the PR for the Session timer in joomla-framework

**Comment by mbabker** [comment]
It's on my list.  Just had a lot going on the last few weeks.

**Comment by anuragteapot** [comment]
@wilsonge can you review these changes.

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 0f3c6a106695baa71f1e897e5dacd04f64ab6a1d<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19324">issues.joomla.org/tracker/joomla-cms/19324</a>.</sub>

**Comment by Quy** [comment]
RTC

**Comment by wilsonge** [comment]
Awesome work! Thanks so much for this series :)

**Comment by anuragteapot** [comment]
@wilsonge  Thanks :)

**Pull request merged**